client/payment-details/timeline/map-events.js

```diff
@@ -223,6 +223,48 @@ const composeFXString = ( event ) => {
 	);
 };
 
+/**
+ * Returns an array containing fee breakup.
+ *
+ * @param {Object} event Event object
+ *
+ * @return {Array} Array of formatted fee strings
+ */
+const feeBreakup = ( event ) => {
+	if ( ! event?.fee_rates?.history ) {
```

Comment:
This looks so smart 😬 Can you let me know what the name of this usage `?.`?

Reply:
This is called optional chaining: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
TIL. That's all what I need. Thank you @naman03malhotra :D